File: app/src/main/java/me/kirimin/mitsumine/model/UserId.java
package me.kirimin.mitsumine.model;

import com.activeandroid.Model;
import com.activeandroid.annotation.Column;
import com.activeandroid.annotation.Table;

@Table(name = "userid")
public class UserId extends Model {

    @Column(name = "word", unique = true)
    public String word;
}

File: app/src/main/java/me/kirimin/mitsumine/_common/domain/model/UserId.kt
package me.kirimin.mitsumine._common.domain.model

import com.activeandroid.Model
import com.activeandroid.annotation.Column
import com.activeandroid.annotation.Table

@Table(name = "userid")
class UserId : Model() {

    @Column(name = "word", unique = true)
    var word: String = ""
}
